### PR TITLE
[FVM] Use embed for cadence scripts in blueprints

### PR DESCRIPTION
**Pull request by janezpodhostnik**
originally from:  https://github.com/onflow/flow-go/pull/2684

- extract cadence scripts into separate cdc files (this way we can late write tests for the cadence scripts or run some CI automation over them)
- change some constants from a (domain string, path string) pair to a cadence path, so there is less code
- change cadence template to use arguments and the template replace function from core-contracts

### CODE REVIEW

**Review by Tonix517 — APPROVED** [review]
LGTM

**Comment by janezpodhostnik** [comment]
converting to draft since I forgot to fix the integration tests.

File: new.txt [hunk 1]
```diff
@@ -0,0 +1,135 @@
+-test.shuffle 1657722833025907000
```
Comment (pattyshack) [inline new.txt:1]: Remove?  Looks like this got accidentally included

**Review by pattyshack — APPROVED** [review]
in the future, could you split this into multiple PRs?  it's easier than reviewing individual commits

**Review by m4ksio — APPROVED** [review]

**Comment by janezpodhostnik** [comment]
bors merge

**Pull request merged**